README.md

```diff
@@ -12,50 +12,121 @@
 
 [![Sauce Test Status](https://saucelabs.com/browser-matrix/skatejs.svg)](https://saucelabs.com/u/skatejs)
 
-Skate is high level, functional abstraction over the web component [specs](https://github.com/w3c/webcomponents) that:
+Skate is a functional abstraction over [the web component standards](https://github.com/w3c/webcomponents) that:
 
 - Produces cross-framework compatible components
 - Abstracts away common attribute / property semantics via `props`, such as attribute reflection and coercion
 - Adds several lifecycle callbacks for responding to prop updates, rendering and more
 - Provides a base set of [mixins](http://justinfagnani.com/2015/12/21/real-mixins-with-javascript-classes/) that hook into renderers such as [@skatejs/renderer-preact](https://github.com/skatejs/renderer-preact).
 
-## Installing
+## Anatomy of a Skate web component
 
-```sh
-npm install skatejs
+At its core, Skate is about creating [Custom Elements](https://w3c.github.io/webcomponents/spec/custom/).
+Skate provides a series of [mixin functions](http://justinfagnani.com/2015/12/21/real-mixins-with-javascript-classes/)
+that enable you to control what your component can do.
+
+For example, to create a simple greeting component:
+
+```html
+<x-hello>Bob</x-hello>
 ```
 
-To use Skate with a renderer, you'll want to install one of them:
+We could define a Skate component that renders the contents of our Custom Element:
 
-```sh
-npm install skatejs @skatejs/renderer-[renderer]
+```js
+import { withComponent, withRenderer } from 'skatejs';
```

Comment:
All you need is `withComponent()`, the `withRenderer()` part is included with it and has no effect when composed aside from adding another prototype to the chain.